openstack/elektra/charts/postgres/templates/backup-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.backup.enabled }}
```

Comment:
Ok, and whats the reason for these two settings being in a configmap instead of environment variables like the rest?